playground/webpack.config.js

```diff
@@ -43,6 +47,9 @@ module.exports = (env, argv) => {
       },
     },
     plugins: [
+      new webpack.DefinePlugin({
+        global: '(typeof globalThis ? globalThis : self)',
```

Comment:
rather than use the [polyfill](https://github.com/webpack/webpack/blob/911ec1aa67011e25aa1449610f5b0b557edd5459/lib/runtime/GlobalRuntimeModule.js#L20-L42) that we are telling webpack to not use above, use this string of code.

When a module is trying to use [`global`](https://nodejs.org/docs/latest/api/globals.html#globals_global), webpack will regex in this string of code. if [globalThis](https://mathiasbynens.be/notes/globalthis) is defined, use it. If it is not defined, use [`self`](https://developer.mozilla.org/en-US/docs/Web/API/Window/self), since it is available in worker and DOM, and is a shorter string than "window".